src/main/webapp/resources/js/pages/analysis/components/AnalysisDetails.jsx

```diff
@@ -0,0 +1,197 @@
+            priorityList.push(
+              <Select.Option key={priority} value={priority}>{priority}</Select.Option>
+            )
+        }
+        return priorityList;
+    }
+
+  return (
+      <>
+          <Tabs defaultActiveKey="4"
+                tabPosition="left"
+                style={{marginLeft:150, paddingTop:25}}
+                animated={false}
+          >
+              <TabPane tab={getI18N("analysis.tab.content.analysis.details")} key="4" style={{minWidth:300}}>
+                  <Col span={12}>
+                    <h2 style={{fontWeight: "bold"}} className="spaced-bottom">{getI18N("analysis.tab.content.analysis.details")}</h2>
+
+                    <Row>
+                      <Col xs={{ span: 12, offset: 0 }} lg={{ span: 12, offset: 0 }}>
+                        <label style={{fontWeight: "bold"}}>{getI18N("analysis.tab.content.analysis.analysis-name")}</label>
+                        <Input size="large" placeholder={analysisContext.analysis.name} id="analysis-name" />
+                      </Col>
+
+                      { analysisDetailsContext.updatePermission ?
+                          <Col xs={{ span: 12, offset: 0 }} lg={{ span: 12, offset: 0 }}>
+                            <Button size="large" type="primary" className="spaced-left__sm spaced-top__lg" onClick={() => updateSubmissionName()}>{getI18N("analysis.tab.content.analysis.update.button")}</Button>
+                          </Col>
+                        : null
+                      }
+                    </Row>
+
+                    { analysisContext.isAdmin && analysisContext.analysisState == "NEW" ?
+                        <Row className="spaced-top">
+                            <Col xs={{ span: 12, offset: 0 }} lg={{ span: 12, offset: 0 }}>
+                                <label style={{fontWeight: "bold"}}>Priority</label>
+                                <Select defaultValue={analysisContext.analysis.priority} className="form-control" onChange={updateAnalysisPriority}>
+                                  {renderPriorities()}
+                                </Select>
+                            </Col>
+                        </Row>
+                        : null
+                    }
+
+                      <div className="spaced-top__lg">
+                      <List
+                        itemLayout="horizontal"
+                        dataSource={analysisDetails}
+                        renderItem={item => (
+                          <List.Item>
+                            <List.Item.Meta
+                              title={<span style={{fontWeight: "bold"}}>{item.title}</span>}
+                              description={<span>{item.desc}</span>}
+                            />
+                          </List.Item>
+                        )}
+                      />
+                      </div>
+                      <hr style={{backgroundColor: "#E8E8E8", height: "1px", border: "0"}} />
```

Comment:
Use [ant.design `Divider`](https://ant.design/components/divider/) instead

Reply:
Fixed in 4d5b63fdf69f55254f7b8ec40e149f8fc26fcd4a